src/components/People/People.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link, useParams } from 'react-router-dom';
+import data from '../../../public/api/people.json';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+const getSlugByName = (name: string) => {
+  const person = data.find(p => p.name === name);
```

Comment:
Consider handling the case where `data` might be undefined or not an array. This could prevent potential runtime errors if the JSON file is not loaded correctly.